### PR TITLE
[UI] Key Helper

### PR DESCRIPTION
**Pull request by TimGoll**
image
_(The full binding set and the description are only shown when the scoreboard is open)_

Video: https://www.youtube.com/watch?v=NzoEboD6slc

Adds binding infos to the screen when a specific binding can be pressed.

- fixed blur rendering cutout
- removed now unnecessary punch-o-meter text
- fixed disguiser hotreload

ToDo
- [x] add settings to F1 menu
- [x] add extra stuff: flashlight, voice, chat (hidden by default), spray
- [x] F2 mute as spec
- [x] quickchat / radiocommands (zoomkey)
- [x] correctly track scoreboard open
- [x] fix lang strings to changed behavior
- [x] add lang strings for bind names
- [x] make sure team chat bind is only shown when chat is possible
- [x] ctrl -> show mouse as spec

### CODE REVIEW

**Review by saibotk — APPROVED** [review]

File: lua/terrortown/lang/de.lua [hunk 1]
```diff
@@ -1897,6 +1893,47 @@ L.label_voice_scaling_mode_linear = "Linear"
...
+--L.label_keyhelper_possession_backward = "prop: backward"
+--L.label_keyhelper_free_roam = "leave object and roam free"
+--L.label_keyhelper_flashlight = "toggle flashlight"
+--L.label_keyhelper_quickchat = "open quickchat"
+--L.label_keyhelper_voice_global = "global voice chat"
+--L.label_keyhelper_voice_team = "team voice chat"
+--L.label_keyhelper_chat_global = "global chat"
+--L.label_keyhelper_chat_team = "team chat"
+--L.label_keyhelper_show_all = "show all"
+--L.label_keyhelper_disguiser = "toggle disguiser"
```
Comment (Histalek) [inline lua/terrortown/lang/de.lua:1935]:
```suggestion
L.help_keyhelp = [[
Tastenhelfer sind ein UI Element, welches dauerhaft relevante Tastenbelegungen für den Spieler anzeigen. Gerade unerfahrene Spieler können hiervon profitieren. Es gibt drei unterschiedliche Kategorien von Tastenbelegungen:

Core: Diese Kategorie enthält die wichtigsten Tastenbelegungen in TTT2. Ohne diese ist es schwer das volle Potential von TTT2 zu nutzen.
Extra: Ähnlich zu 'core', enthält aber eher nicht durchgehend nötige Tastenbelegungen. Dinge wie der Chat, Sprachchat und Taschenlampe sind hier enthalten. Gerade für neue Spieler kann das aktivieren sinnvoll sein.
Equipment: Ausrüstungsgegenstände können eigene Tastenbelegungen haben, diese werden in dieser Kategorie angezeigt.

Deaktivierte Kategorien werden weiterhin angezeigt solange die Punktetafel geöffnet ist]]

L.label_keyhelp_show_core = "Aktiviere dauerhaftes anzeigen der 'core' Tastenhelfer"
L.label_keyhelp_show_extra = "Aktiviere dauerhaftes anzeigen der 'extra' Tastenhelfer"
L.label_keyhelp_show_equipment = "Aktiviere dauerhaftes anzeigen der 'equipment' Tastenhelfer"

L.header_interface_keys = "Tastenhelfer Einstellungen"
L.header_interface_wepswitch = "Waffenwechsel UI Einstellungen"

L.label_keyhelper_help = "öffne Spielmodus Menü"
L.label_keyhelper_mutespec = "Wechsle Zuschauer Sprachmodus"
L.label_keyhelper_shop = "öffne Ausrüstungsshop"
--L.label_keyhelper_show_pointer = "free mouse pointer"
L.label_keyhelper_possess_focus_entity = "fokusiertes Objekt übernehmen"
L.label_keyhelper_spec_focus_player = "fokusiertem Spieler zuschauen"
L.label_keyhelper_spec_previous_player = "vorheriger Spieler"
L.label_keyhelper_spec_next_player = "nächster Spieler"
L.label_keyhelper_spec_player = "Spieler zuschauen"
L.label_keyhelper_possession_jump = "Objekt: springen"
L.label_keyhelper_possession_left = "Objekt: links"
L.label_keyhelper_possession_right = "Objekt: rechts"
L.label_keyhelper_possession_forward = "Objekt: vorwärts"
L.label_keyhelper_possession_backward = "Objekt: rückwärts"
L.label_keyhelper_free_roam = "verlasse Objekt und bewege dich frei"
L.label_keyhelper_flashlight = "Taschenlampe an-/ausschalten"
L.label_keyhelper_quickchat = "Schnellchat öffnen"
L.label_keyhelper_voice_global = "Globaler Sprachchat"
L.label_keyhelper_voice_team = "Team Sprachchat"
L.label_keyhelper_chat_global = "Globaler Chat"
L.label_keyhelper_chat_team = "Team Chat"
L.label_keyhelper_show_all = "Alle anzeigen"
L.label_keyhelper_disguiser = "Tarnung an-/ausschalten"
```

Might not be the best translation, but better than nothing.

Also i'm not sure what is meant by "free mouse pointer" so i left that one out.

**Review by Histalek — CHANGES_REQUESTED** [review]
Nice addition overall :)

File: lua/terrortown/lang/de.lua (hunk 1 above)
Comment (TimGoll) [reply lua/terrortown/lang/de.lua:1935]: you literally free it. If you press ctrl, you can move your mouse like in a 2D menu

**Comment by TimGoll** [comment]
done @Histalek

**Review by Histalek — APPROVED** [review]

**Pull request merged**